gips/data/cdl/cdl.py

```diff
@@ -67,29 +80,48 @@ class cdlData(Data):
     _products = {
         _cdl: {
             'description': 'Crop Data Layer',
-            'assets': [_cdl],
+            'assets': [_cdl, _cdlmkii],
             'bands': [{'name': _cdl, 'units': 'none'}],
             # presently 'startdate' & 'latency' are permitted to be unspecified
             # by DH gips.utils.get_data_variables
         }
     }
 
-    _legend = None
+    def process(self, products, overwrite=False, **kwargs):
```

Comment:
Tom's right on here.  Though the probability of anything going awry is low here, it is probably good practice for the tempdir manager to be used here.  `gips.data.core.Data.proc_temp_dir_manager` is the deco for it.